pages/index.js

```diff
@@ -1,54 +1,81 @@
 import styled from "styled-components";
 import { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function Landingpage() {
   const [output, setOutput] = useState("");
+  const [isClicked, setIsClicked] = useState(false);
+  const router = useRouter();
+
   const handleSubmit = (event) => {
     event.preventDefault();
+
     const entry = event.target.elements.text.value;
     setOutput(entry);
+    event.target.reset();
+  };
+
+  const onLinkClick = (event) => {
+    event.preventDefault();
+    setIsClicked(true);
   };
+
   return (
     <>
       <main>
-        <form onSubmit={handleSubmit}>
-          <StyledFieldset>
-            <StyledLegend>Where is your journey going?</StyledLegend>
-            <StyledLabel htmlFor="text">City or Country</StyledLabel>
-            <input
-              type="text"
-              required
-              id="text"
-              name="text"
-              pattern="^[^\sa0-9].*$"
-              maxLength="20"
-              placeholder="enter your destination..."
-            />
-            <StyledButton type="submit">Submit</StyledButton>
-            <p>{output}</p>
-          </StyledFieldset>
-        </form>
+        <StyledForm onSubmit={handleSubmit}>
+          <legend>Where is your journey going?</legend>
+          <label htmlFor="text">City or Country?</label>
+          <StyledInput
+            type="text"
+            required
+            id="text"
+            name="text"
+            pattern="^[^\sa0-9].*$"
+            maxLength="33"
+            placeholder="enter your destination..."
+            inputColor="rebeccapurple"
+          />
+          <StyledButton type="submit">start your journey</StyledButton>
+          <StyledLink
+          href="addNewEntryPage"
+          className={router.pathname === "/addNewEntryPage"}
```

Comment:
is this doing something I miss? It shouldnt.

Reply:
there was one function missing, which is running now. if you click on the displayed city/countryname, it is displayed on the next "new entry" page. 
it's not very clear, thats true

Reply:
`lassName={router.pathname === "/addNewEntryPage"}`

this line should serve no purpose.